src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,96 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieData } from '../../types/MovieData';
+import { Movie } from '../../types/Movie';
+import { ResponseError } from '../../types/ReponseError';
+import { MovieCard } from '../MovieCard';
+import { IMDB_URL, IMG_URL } from '../../constants';
+
+type Props = {
+  addMovie: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [response, setResponse] = useState<MovieData | ResponseError>();
+  const [movie, setMovie] = useState<Movie>();
+  const [isError, setIsError] = useState<boolean>();
+  const [isSearched, setIsSearched] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isDisabled, setIsDisabled] = useState(true);
+
+  const sendRequest = (value: string) => {
+    getMovie(value)
+      .then(res => {
+        setResponse(() => res);
```

Comment:
```suggestion
        setResponse(res);
```
if you don't need the current state to set the new one, use common `setState`, instead of functional `setState`